.github/workflows/pr.yaml

```diff
@@ -0,0 +1,14 @@
+name: Run tests
+on: pull_request
+defaults:
+  run:
+    working-directory: matching-game
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install modules
+        run: npm i
+      - name: Run tests
+        run: npm run test
```

Comment:
these are the steps of the workflow. basically, check out the code from GitHub, install the JS dependencies in package.json, then run the tests.